### PR TITLE
Make MaxPollingInterval configurable for each queue processor

### LINKED ISSUE #1030: Make MaxPollingInterval configurable for each queue processor
In some cases there is a need to poll queues with different latencies (e.g. chat queue with low latency or image processing queue with moderate latency). In such cases one global MaxPollingInterval makes it impossible to maintain only the needed amount of polling, i.e. it leads to increased storage costs.

### PR DESCRIPTION
**Pull request by Dia6lo**
Fixes #1030

### CODE REVIEW

File: src/Microsoft.Azure.WebJobs.Host/Queues/Listeners/HostMessageListenerFactory.cs [hunk 1]
```diff
@@ -83,16 +80,9 @@ public Task<IListener> CreateAsync(CancellationToken cancellationToken)
         {
             ITriggerExecutor<IStorageQueueMessage> triggerExecutor = new HostMessageExecutor(_executor, _functionLookup, _functionInstanceLogger);
 
-            TimeSpan configuredMaximum = _queueConfiguration.MaxPollingInterval;
-            // Provide an upper bound on the maximum polling interval for run/abort from dashboard.
```
Comment (mathewc) [inline src/Microsoft.Azure.WebJobs.Host/Queues/Listeners/HostMessageListenerFactory.cs]: So you're removing this upper bound here. Previously we were bounding this. Might not be an issue - what was your thought here?

**Comment by mathewc** [comment]
Merged, thanks :)

File: src/Microsoft.Azure.WebJobs.Host/Queues/Listeners/HostMessageListenerFactory.cs (hunk 1 above)
Comment (Dia6lo) [inline src/Microsoft.Azure.WebJobs.Host/Queues/Listeners/HostMessageListenerFactory.cs]: Yeah, I see now the point of this maximum - without it when we set high MaxPollingInterval, Dashboard become less responsive. I think that complete removal of IDelayStrategy injection is a mistake. We can bring it back and create new strategy only if none was passed. Second option (and I think the best option) is to pass custom QueueProcessor in this case. Another option, that would be great too (for extensibility), is adding custom IDelayStrategy to QueueProcessor itself and fetch it in QueueListener constructor (but it's sort of completely separate feature).

Comment (Dia6lo) [inline src/Microsoft.Azure.WebJobs.Host/Queues/Listeners/HostMessageListenerFactory.cs]: Anyway, one of these options should be applied, or there will be troubles in the future.

Comment (mathewc) [inline src/Microsoft.Azure.WebJobs.Host/Queues/Listeners/HostMessageListenerFactory.cs]: Fixed

**Pull request merged**